package.json

```diff
@@ -33,7 +34,7 @@
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "git diff --name-only --cached --relative | egrep .js$ | xargs --no-run-if-empty standard"
+    "lint": "node_modules/.bin/standard"
```

Comment:
nah, that `gif diff` madness is not mine :) and I already found times that super fast linting check (with `git diff`) didn't caught linting errors that lead to mem leaks in another project. I recommend running standard and let it check everything.